draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1842,7 +1842,7 @@ An authentication tag may verify under multiple keys, nonces, or associated data
 
 AEGIS is fully committing in the restricted setting where an adversary cannot control the associated data. As shown in {{IR23}}, with the ability to alter the associated data, it is possible to efficiently find multiple keys that will verify the same authenticated ciphertext.
 
-Protocols mandating a fully committing scheme without that restriction can provide the associated data as input to a cryptographic hash function and use the output as the `ad` parameter of the `Encrypt` and `Decrypt` functions. The selected hash function must ensure a minimum of 128-bit preimage resistance. An instance of such a function is SHA-256 {{!RFC6234}}.
+Protocols mandating a fully committing scheme without that restriction can provide the associated data as input to a cryptographic hash function and use the output as the `ad` parameter of the `Encrypt` and `Decrypt` functions. For AEGIS-128L and AEGIS-128X, the selected hash function must ensure a minimum of 128-bit collision and preimage resistance. An instance of such a function is SHA-256 {{!RFC6234}}.
```

Comment:
How come only AEGIS-128L and AEGIS-128X? Do you mean because of their 128-bit security level?

I think we can say 128-bit for all of them since newer algorithms have started targeting that rather than 256-bit anyway and that's what the commitment literature recommends.

Reply:
I originally tried to address your comment in a different way, changed it back, and ended up with that nonsense making more harm than good :)

Reverted.